api/tests/DataPersister/ActivityDataPersisterTest.php

```diff
@@ -35,27 +33,7 @@ protected function setUp(): void {
         $categoryRoot->contentType->name = 'ColumnLayout';
         $this->activity->category->setRootContentNode($categoryRoot);
 
-        $this->dataPersister = new ActivityDataPersister($this->dataPersisterObservable);
-    }
-
-    public function testDelegatesSupportCheckToDecorated() {
```

Comment:
Diesen und den nächsten Test würde ich nicht entfernen (für alle DataPersisters), oder zumindest in einem vereinfachten Rahmen drin lassen.

So testen wir, dass der constructor von `ActivityDataPersister` auch die richtige Klasse an den parent übergibt. Jetzt könnte ich den constructor auch wie folgt implementieren, und der Test läuft trotzdem durch:

```php
public function __construct(
        DataPersisterObservable $dataPersisterObservable
    ) {
        parent::__construct(
            'Dummy',
            $dataPersisterObservable,
        );
    }
```

Reply:
Drei tests duplizieren um zu testen dass ein string richtig ist finde ich bisschen overkill.
Vor allem da man hoffentlich Logik in die persister packt, die dann in den API Tests getestet werden (Womit auch das richtige setzen des Strings getestet ist).

Reply:
Du, ich will's nicht unnötig verkomplizieren.

Müssten dann einfach sicherstellen, dass wir auch zu jedem Endpunkt mindestens 1 API-Test haben, der fehlschlägt, wenn der DataPersister nicht richtig verdrahtet ist.

Zumindest die Tests für `SingleText` und `Storyboard` laufen heute noch problemlos durch, auch wenn der DataPersister nicht anspringt.

Reply:
Habe ich zumindest auf diesem branch getestet. Habe überall "blabla" als Klasse angegeben, und sie failen.
Zum glück testen wir die response vom create so genau.

![grafik](https://user-images.githubusercontent.com/1506818/149211145-6beaccb2-1334-4f08-a9b0-176debf01d61.png)